Formula/polynote.rb

```diff
@@ -0,0 +1,47 @@
+class Polynote < Formula
+  desc "The polyglot notebook with first-class Scala support"
+  homepage "https://polynote.org/"
+  url "https://github.com/polynote/polynote/releases/download/0.3.0/polynote-dist.tar.gz"
+  version "0.3.0"
```

Comment:
This is not needed with the current URL.

Reply:
If I take it out, I get this audit error:
```
polynote:
  * Stable: version (dist) is set to a string without a digit
```
Doesn't seem to me the version is properly extracted from the formula's URL:
```
polynote: stable dist
The polyglot notebook with first-class Scala support
https://polynote.org/
[...]
```
Should I keep the `version` declaration, or is there a different strategy to follow?

Reply:
Keep it then